stable/python/Dockerfile.3.6

```diff
@@ -17,9 +17,9 @@ RUN python ./get-pip.py
 RUN pip install bottle==0.12.13 cherrypy==8.9.1 wsgi-request-logger prometheus_client
 
 WORKDIR /
-ADD kubeless.py .
+ADD _kubeless.py .
```

Comment:
Renaming this file to something that has a very low likelihood of name collision with a user-defined module.
For example, the user can provide `--from-file kubeless.py --handler kubeless.handler` and it would still work.
This wasn't required when we were using `fork`, but is required when we use `forkserver`.